src/gnome_abrt/dbus_problems.py

```diff
@@ -148,7 +148,7 @@ def chown_problem(self, problem_id):
             return True
         except dbus.exceptions.DBusException as ex:
             logging.warning(
-                    _("Can't chown problem '{0}' over DBus service: {1!s}")
+                    _("Can’t chown problem “{0}” over D-Bus service: {1!s}")
```

Comment:
Just commenting: this string doesn’t sound easily translatable, as it uses “chown” as a verb.